src/splitInteger.test.js

```diff
@@ -4,16 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-  expect(splitInteger(17, 4))
-    .toHaveLength(4);
+  expect(splitInteger(10, 2)).toEqual([5, 5]);
+});
+
+test(`should return a part equals to a value
+  when splitting into 1 part`, () => {
+  expect(splitInteger(15, 1)).toEqual([15]);
```

Comment:
The test description does not match the test implementation. The description talks about returning a part equals to a value when splitting into 1 part, but the test checks for the specific case of 15 divided into 1 part.

Reply:
> The test description does not match the test implementation. The description talks about returning a part equals to a value when splitting into 1 part, but the test checks for the specific case of 15 divided into 1 part.

Sorry, already fixed my mistakes